src/mavsdk.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_FULL_BINDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+
+Name: libmavsdk
+Description: API and library for MAVLink compatible systems written in C++17
+Version: @MAVSDK_VERSION_STRING@
+Requires.private: libcurl jsoncpp tinyxml2
```

Comment:
So this is only for the library without mavsdk_server, right?

Reply:
Yes. I don't know how to manage the option to be honest. Maybe have two templates and have CMake choose depending on `BUILD_MAVSDK_SERVER`?

But anyway it's the first time I write a pc file, I thought that keeping it simple would make sense :sweat_smile: